setup.go

```diff
@@ -24,10 +24,20 @@ func setup(c *caddy.Controller) error {
 	f := New(t)
 
 	for c.Next() {
-		var rcode string
+		var (
+			original bool
+			rcode    string
+		)
 		if !c.Dispenser.Args(&rcode) {
 			return c.ArgErr()
 		}
+		if rcode == "ORIGINAL" {
```

Comment:
Why all caps?

Reply:
Wanted to be consistent with rcode constant. But lowercase seems better here. Changed.

Reply:
Thanks, yes, I think lower is better, since this *isn't* an `rcode`.